cmd/auth/main.go

```diff
@@ -35,6 +36,7 @@ import (
 	"github.com/caarlos0/env/v10"
 	"github.com/jmoiron/sqlx"
 	chclient "github.com/mainflux/callhome/pkg/client"
+	mflog "github.com/mainflux/mainflux/logger"
```

Comment:
instead of using the mainflux logger you could use the kitlogger you have
```suggestion
	mflog "github.com/absmach/magistrala/kitlogger"
```

Reply:
you can do this for the rest